src/analytics/private/refresh-materialized-views-workflow.ts

```diff
@@ -250,6 +316,33 @@ export class RefreshMaterializedViewsWorkflow extends Construct {
     return fn;
   }
 
+  private checkWorkflowStartFn(props: RefreshMaterializedViewsWorkflowProps, logGroup: LogGroup): IFunction {
+    const fnSG = props.securityGroupForLambda;
+
+    const fn = new SolutionNodejsFunction(this, 'CheckRefreshWorkflowStart', {
+      entry: join(
+        this.lambdaRootPath,
+        'check-refresh-workflow-start.ts',
+      ),
+      handler: 'handler',
+      memorySize: 128,
+      timeout: Duration.minutes(3),
+      logConf: {
```

Comment:
it's useless. remove it if it's an optional argument

Reply:
refactor it